hubstorage/job.py

```diff
@@ -7,12 +7,10 @@
 
 class Job(object):
 
-    def __init__(self, client, key, auth=None, jobauth=None, metadata=None):
+    def __init__(self, client, key, auth=None, metadata=None):
```

Comment:
jobauth is still useful and it must be set for jobs returned by calls to `HubstorageClient.start_job()`